src/patterns/letter-from-the-ceo/frontend.scss

```
@@ -0,0 +1,51 @@
+.wmf-pattern-letter-from-the-ceo {
+	padding-left: 1.25rem;
+	padding-right: 1.25rem;
+
+	> .wp-block-group__inner-container {
+		max-width: 646px;
+    	margin: 0 auto;
+	}
+
+	.wp-block-quote {
+		margin: 1em 0;
+		border: 0;
+		padding-left: 0;
+
+		p {
+			font-family: 'Noto Sans';
+			font-size: 28px;
+			line-height: 34px;
+			font-weight: bold;
+			font-style: normal;
+		}
+	}
+
+	.profile {
+		display: grid;
+		grid-template-columns: 110px 1fr;
+		width: auto;
+		align-items: center;
+		padding: 0.5em;
+		border: 0;
+		margin-bottom: 1em;
+		border: 0;
+
+		&:hover {
+			border: 0;
+		}
+	}
+
+	.profile__image {
+		margin-bottom: 0;
+		width: 80px;
+		height: 80px;
+	}
+
+	.profile__content {
+		.profile__name {
+			font-size: 14px;
+			line-height: 15px;
```

We should be using `rem` for font size, not pixels. Conveniently the foundation and endowment appear to use the same `16px` base font size, so this would be
```suggestion
			font-size: 0.875rem;
			line-height: 1.07;
```
[Line-height](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height) should usually be unitless, and is treated as a multiple of the fontsize.